### PR TITLE
Using PayPalMessageConfig we set the integration name and version to support Messaging Analytics

### PR DESCRIPTION
**Pull request by warmkesselj**
### Summary of changes

Using PayPalMessageConfig we set the integration name and version to support Messaging Analytics

### Checklist

- [ ] Added a changelog entry

### Authors
@saperi22 
@jwarmkessel 
@KunJeongPark

### CODE REVIEW

File: Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift [hunk 1]
```diff
@@ -52,6 +58,8 @@ public class BTPayPalMessagingView: UIView {
                 self.notifyFailure(with: BTPayPalMessagingError.payPalClientIDNotFound)
                 return
             }
+            
+
```
Comment (jaxdesmarais) [inline Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift:62]:
```suggestion
```

File: Sources/BraintreeCore/BTCoreConstants.swift [hunk 2]
```diff
@@ -27,4 +27,9 @@ import Foundation
     /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
     /// Key for userInfo dictionary that contains the BTJSON body of the HTTP error response
     public static let jsonResponseBodyKey: String = "com.braintreepayments.BTHTTPJSONResponseBodyKey"
+    
+    /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    /// Named used for identifiying the integrating SDK e.g. BTPayPalMessageView
+    public static let braintreeSDKIdentifier: String = "BT_SDK"
```
Comment (jaxdesmarais) [inline Sources/BraintreeCore/BTCoreConstants.swift:33]: I see in the Android PR we just hardcode this value and here we have it extracted out. I don't think we will use this for anything but messaging so probably fine to just hardcode where we call `setGlobalAnalytics`

File: Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift [hunk 3]
```diff
@@ -36,6 +36,12 @@ public class BTPayPalMessagingView: UIView {
     /// - Parameter request: an optional `BTPayPalMessagingRequest`
     /// - Warning: use `BTPayPalMessagingDelegate` protocol to receive notifications for events
     public func start(_ request: BTPayPalMessagingRequest = BTPayPalMessagingRequest()) {
+
```
Comment (jaxdesmarais) [inline Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift:39]:
```suggestion
```

File: Sources/BraintreeCore/BTCoreConstants.swift (hunk 2 above)
Comment (KunJeongPark) [reply Sources/BraintreeCore/BTCoreConstants.swift:33]: Yes we had a discussion about that, we didn't want to make a typo in any subsequent changes, but I'm ok with hardcoding.

**Review by scannillo — APPROVED** [review]

**Review by jaxdesmarais — APPROVED** [review]

**Pull request merged**